doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -190,7 +190,7 @@ The procedure to install CORTX on OVA is mentioned below.
    
 #. Using the management IP from the **ip a l** command,  refer to these instructions to `configure the CORTX GUI <Preboarding_and_Onboarding.rst>`_. 
 
-#. Now that you have the complete system up and running, using the data IP from the **ip a l** command, use these instructions `to test the system <Performing_IO_Operations_Using_S3Client.rst>`_  and observe activity in the GUI.  For example, the below picture shows a CORTX dashboard after a user did an *S3 put* followed by an *S3 get*.
+#. Now that you have the complete system up and running, using the data IP from the **ip a l** command, use these instructions `to test the system <testing_io.rst>`_  and observe activity in the GUI.  For example, the below picture shows a CORTX dashboard after a user did an *S3 put* followed by an *S3 get*.
```

Comment:
@TechWriter-Mayur  Do we need to revert this also. I see [Performing_IO_Operations_Using_S3Client.rst](https://github.com/TechWriter-Mayur/cortx/blob/main/doc/Performing_IO_Operations_Using_S3Client.rst)  is also present